### PR TITLE
Special quests

### PR DESCRIPTION
**Pull request by nheyek**

### CODE REVIEW

File: app/src/main/java/ch/epfl/sweng/studyup/firebase/Firestore.java [hunk 1]
```diff
@@ -187,6 +192,10 @@ public void addQuestion(final Question question) {
         db.collection(FB_QUESTIONS).document(question.getQuestionId()).set(questionData);
     }
 
+    public void updateQuests(Map<String, Object> remotePlayerData) {
```
Comment (JulesssG) [inline app/src/main/java/ch/epfl/sweng/studyup/firebase/Firestore.java]: Is it an oversight or wanted?

File: app/src/main/java/ch/epfl/sweng/studyup/utils/adapters/QuestListViewAdapterStudent.java [hunk 2]
```diff
@@ -0,0 +1,54 @@
+package ch.epfl.sweng.studyup.utils.adapters;
```
Comment (JulesssG) [inline app/src/main/java/ch/epfl/sweng/studyup/utils/adapters/QuestListViewAdapterStudent.java]: This is an old file not used anymore (it's a nested class now), it should be deleted

File: app/src/main/java/ch/epfl/sweng/studyup/utils/adapters/SpecialQuestListViewAdapter.java [hunk 3]
```diff
@@ -0,0 +1,64 @@
+package ch.epfl.sweng.studyup.utils.adapters;
```
Comment (JulesssG) [inline app/src/main/java/ch/epfl/sweng/studyup/utils/adapters/SpecialQuestListViewAdapter.java]: Good call on making this package, but why not putting all adapters in there? (listItemAdapter, ViewPagerAdapter)

**Comment by Gorzen** [comment]
7

**Pull request closed without merging**